.github/workflows/submoduleNotifyParent.yml

```
@@ -0,0 +1,30 @@
+name: Send submodule updates to parent repo
+
+on:
+  push:
+    branches: 
+      - main
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          repository: org/parent_repository
```

This seems like a placeholder, should change to the proper repo name

```suggestion
          repository: osu-atri/osu-dictionary
```

---

Just forgot to change that, thx